cli/lsp/config.rs

```diff
@@ -1617,10 +1625,10 @@ impl ConfigTree {
       return true;
     }
     self
-      .root
+      .scopes
       .lock()
-      .as_ref()
-      .is_some_and(|(_, d)| d.watched_files.contains_key(specifier))
+      .values()
+      .any(|data| data.watched_files.contains_key(specifier))
```

Comment:
Not sure how many scopes we expect to have, but we may want to optimize this down the line